include/xten/Conversion/ATenToXTen.td

```diff
@@ -26,18 +26,21 @@ def : Pat<(Torch_AtenReluOp (XTen_Conv2dOp $a,$b,$c,$d,$e,$f,$g)),
 def : Pat<(Torch_AtenRelu_Op (XTen_Conv2dOp $a,$b,$c,$d,$e,$f,$g)),
           (XTen_Conv2dReLUOp $a,$b,$c,$d,$e,$f,$g)>;
 
-// def : Pat<(aten_LeakyReluOp (XTen_Conv2dOp $a,$b,$c,$d,$e,$f,$g),$h),
-//           (XTen_Conv2dLReLUOp $a,$b,$c,$d,$e,$f,$g,$h)>;
+def : Pat<(Torch_AtenLeakyReluOp (XTen_Conv2dOp $a,$b,$c,$d,$e,$f,$g),$h),
+          (XTen_Conv2dLReLUOp $a,$b,$c,$d,$e,$f,$g,$h)>;
+
+def : Pat<(Torch_AtenLeakyRelu_Op (XTen_Conv2dOp $a,$b,$c,$d,$e,$f,$g),$h),
+          (XTen_Conv2dLReLUOp $a,$b,$c,$d,$e,$f,$g,$h)>;
```

Comment:
If there are really 2 spellings for "Torch_AtenLeakyRelu?Op", then should there just be a pattern that canonicalizes one into the other?

Reply:
For many ops there is an 'op_' version which indicates in-place computation.  So they really are two different ops.